### PR TITLE
Change code block of traefik-web-ui to match file

### PR DESCRIPTION
**Pull request by drewgwallace**
@dtomcej Docs still show corrections necessary against Master
https://github.com/containous/traefik/blob/master/examples/k8s/ui.yaml

### CODE REVIEW

File: docs/user-guide/kubernetes.md [hunk 1]
```diff
@@ -323,6 +323,7 @@ For more information, check out [the documentation](https://github.com/kubernete
 Lets start by creating a Service and an Ingress that will expose the [Træfik Web UI](https://github.com/containous/traefik#web-ui).
 
 ```yaml
+---
```
Comment (dtomcej) [inline docs/user-guide/kubernetes.md]: This line is not needed.

File: docs/user-guide/kubernetes.md [hunk 2]
```diff
@@ -332,24 +333,24 @@ spec:
...
   rules:
   - host: traefik-ui.minikube
     http:
       paths:
-      - backend:
+      - path: /
+        backend:
           serviceName: traefik-web-ui
-          servicePort: 80
+servicePort: web
```
Comment (dtomcej) [inline docs/user-guide/kubernetes.md]: Indenting

File: docs/user-guide/kubernetes.md (hunk 1 above)
Comment (dtomcej) [reply docs/user-guide/kubernetes.md]: Can probably remove that line from the actual yaml too

**Comment by drewgwallace** [comment]
I updated with those changes. It appears this document has the leading '---' on any code block with multiple components.

File: docs/user-guide/kubernetes.md [hunk 3]
```diff
@@ -332,24 +332,24 @@ spec:
...
     targetPort: 8080
 ---
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: traefik-web-ui
   namespace: kube-system
-  annotations:
-    kubernetes.io/ingress.class: traefik
 spec:
```
Comment (timoreimann) [inline docs/user-guide/kubernetes.md]: The annotation makes sense to have in general. I'm thinking that we should rather add it to the YAML file than remove it from the inline spec.

@dtomcej WDYT?

Comment (dtomcej) [reply docs/user-guide/kubernetes.md]: The argument is that we don't specify an ingress class to watch on any of our examples. If we keep it, we _should_ be adding the class to our deployment/daemonset...

Comment (timoreimann) [reply docs/user-guide/kubernetes.md]: Gotcha. Even if we decided to do that, it should be done by a separate PR. No reason to block this one.

**Pull request merged**